src/index.ts

```diff
@@ -33,12 +33,18 @@ const getFiltersFromInput = (): Filters => {
   const ignoreDrafts = getInput("ignoreDrafts") ? getBooleanInput("ignoreDrafts") : true;
 
   let requiredLabels: string[] = [];
-  const labels = getInput("requiredLabels");
+  let ignoredLabels: string[] = [];
+  let labels = getInput("requiredLabels");
   if (labels) {
     requiredLabels = labels.split(",");
   }
 
-  return { daysStale, noReviews, ignoreDrafts, requiredLabels };
+  labels = getInput("ignoredLabels");
+  if (labels) {
+    ignoredLabels = labels.split(",");
```

Comment:
I don't really know if there can be an issue if the value is re-assigned using `getInput`, so maybe it's better to simply declare a new label?

```suggestion
  let ignoredLabelsInput = getInput("ignoredLabels");
  if (ignoredLabelsInput) {
    ignoredLabels = ignoredLabelsInput.split(",");
```

Reply:
yep, at least that looks more confusing

```js
const requiredLabelsInput = getInput..
const ignoredLabelsInput = getInput..
```

would be more clear imo